submission/notifications/src/main/kotlin/ebi/ac/uk/notifications/integration/model/NotificationTemplate.kt

```diff
@@ -7,7 +7,7 @@ open class NotificationTemplate<T : NotificationTemplateModel>(private var templ
     /**
      * Obtaining the string representation by replacing model properties into template string.
      */
-    fun getContent(model: T): String {
+    open fun getContent(model: T): String {
```

Comment:
why we now required to override this?  it is the only function in this base class so for me overriding means that we do not want to sue this one.